### PR TITLE
add: eth and token bridges

### LINKED ISSUE #39: Explain the need to deposit ETH to pay for fees
Our L1<>L2 docs don't currently explain that users will need to deposit ETH in order to pay for fees on L2: https://community.optimism.io/docs/developers/integration.html#bridging-l1-and-l2

We just need to add a brief explanation of the standard ETH bridge and the flow that users will need to take to be able to pay fees.

### PR DESCRIPTION
**Pull request by platocrat**
Fixes #39 and #24.

`Token Bridges` section could use some more love, but I think it's best to flesh it out with additional PRs.

I also made some diagrams :p

### CODE REVIEW

**Comment by platocrat** [comment]
Getting a bunch of errors on this file:
`assets/contractflow-OVM_L1ETHGateway.png `.

will fix soon

**Comment by platocrat** [comment]
Issue was an incorrect letter casing (which Vue somehow still was able to the render locally):
*  Previously --> `contractflow-OVM_l1ETHGateway.png`
* Corrected --> `contractflow-OVM_L1ETHGateway.png`

Checks are taking awhile to update, but I think the last commit fixed the issue.

**Comment by platocrat** [comment]
The merge and the following commit both should've failed. 
You can see the merge commit failed when you check Netlify.

GH is being weird and not updating the checks tho.

File: src/docs/developers/integration.md [hunk 1]
```diff
@@ -66,7 +66,12 @@ If you're using another testing suite like [truffle](https://www.trufflesuite.co
 - [`@eth-optimism/plugins`](https://www.npmjs.com/package/@eth-optimism/plugins): exports "OVM-ified" `waffle.mockProvider` and `ganache` packages which will work with contracts output by the compiler.
 
 An example of usage with [waffle](https://getwaffle.io) can be found in [this great tutorial](https://github.com/ScopeLift/ovm-uniswap-v2-core#porting-solidity-contracts-to-optimism-a-guide-using-uniswap-v2) by [Scopelift](https://www.scopelift.co/) which walks through getting Uniswap V2 ported over.
-If you are using [truffle](https://www.trufflesuite.com/), [here is an example config file](https://github.com/ethereum-optimism/optimism-monorepo/blob/6b535ab759aa2d4bf9325d40ea68aa5f7fc466a6/packages/ovm-toolchain/test/config/truffle-config.js) which shows how to incorporate the compiler and `ganache` for the OVM.
+
+<!--           DEAD LINK!  Should we remove?
+
+If you are using [truffle](https://www.trufflesuite.com/), [here is an example config file](https://github.com/ethereum-optimism/optimism-monorepo/blob/master/packages/ovm-toolchain/test/config/truffle-config.js) which shows how to incorporate the compiler and `ganache` for the OVM.
```
Comment (maurelian) [inline src/docs/developers/integration.md:72]: Someone might know where it is now.

**Review by maurelian — APPROVED** [review]
LGTM

File: src/docs/developers/integration.md (hunk 1 above)
Comment (platocrat) [reply src/docs/developers/integration.md:72]: This comment will be resolved in #58

**Pull request merged**